subprojects/packagefiles/pcre/meson.build

```diff
@@ -80,24 +149,22 @@ char_tables = custom_target('chartables',
   output : 'pcre_chartables.c',
   command : [dftables, '@OUTPUT@'])
 
-pcre8 = build_target('pcre',
+pcre8 = library('pcre',
   pcre8_sources, char_tables,
-  c_args : compile_args,
-  target_type : libtype,
-  include_directories : conf_inc)
+  c_args: compile_args,
+)
 
-pcre_posix = build_target('pcreposix',
+pcre_posix = library('pcreposix',
   'pcreposix.c',
-  c_args : compile_args,
-  target_type : libtype,
-  link_with : pcre8,
-  include_directories : conf_inc)
+  c_args: compile_args,
+  link_with: pcre8,
+)
 
 pcretest = executable('pcretest',
   'pcretest.c', 'pcre_printint.c',
-  c_args : compile_args,
-  link_with : [pcre8, pcre_posix],
-  include_directories : conf_inc,
+  c_args: compile_args,
+  dependencies: libedit_dep,
```

Comment:
Does it build if libedit_dep is not found?

Reply:
It does yes.